news/_posts/2024-02-24-open-door.md

```diff
@@ -1,9 +1,10 @@
 ---
 layout: news
 title: "Lab members participate in ASU Open Door 2024"
-author: "Navish Wadhwa"
-author_handle: "navish"
+author: "Shabduli Sawant"
+author_handle: "shabduli"
 # image: /assets/images/news/microscope.jpg
 category: news
 tags: [people]
 ---
+![Highlights of Open Door 2024](/assets/images/news/Open-Door-highlights-2024-Carolina-Gogerty.pdf)
```

Comment:
I think this way of linking does not work. I suppose we have two options. Easy option, use the "href" function to link to the file and use target="_blank" to open in a new tab. The harder option is to embed a pdf viewer, like done on this website: https://www.materials-theory.group/